coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -331,7 +331,6 @@ public DevelopmentProposal removeFromHierarchy(DevelopmentProposal childProposal
             hierarchyProposal.setHierarchyOriginatingChildProposalNumber(lowestProposalNumber);
             dataObjectService.save(childProposal);
             dataObjectService.save(hierarchyProposal);
-            synchronizeAllChildren(hierarchyProposal);
             return childProposal;
```

Comment:
No. I think thats ok. I think thats the way the code was originally written, but you are correct that it shouldn't be necessary to do this.